docs/source/bird-rr-config.rst

```diff
@@ -0,0 +1,149 @@
+Configuring BIRD as a BGP Route Reflector
+=========================================
+
+These instructions will take you through installing BIRD as a BGP route
+reflector, and updating your other BIRD instances to speak to your new route
+reflector.
+
+These instructions are valid for both Ubuntu 14.04 and RHEL 7.
+
+Prerequisites
+-------------
+
+Before starting this you will need the following:
+
+- A machine running either Ubuntu 14.04 or RHEL 7.
+- SSH access to the machine.
+
+Installation
+------------
+
+Step 1: Install BIRD
+~~~~~~~~~~~~~~~~~~~~
+
+Ubuntu 14.04
+^^^^^^^^^^^^
+
+Add the official `BIRD`_ PPA. This PPA contains fixes to BIRD that are not yet
+available in Ubuntu 14.04. To add the PPA, run::
+
+    sudo add-apt-repository ppa:cz.nic-labs/bird
+
+Once that's done, update your package manager and install BIRD::
+
+    sudo apt-get update
+    sudo apt-get install bird
+
+
+.. _BIRD: http://bird.network.cz/
+
+RHEL 7
+^^^^^^
+
+First, install EPEL. Depending on your system, the following command may be
+sufficient::
+
+    sudo yum install epel-release
+
+If that fails, try the following instead::
```

Comment:
It was unclear to me when reading, but it's possible the above line only works on CentOS.